storage/object/how-to/transfer-bucket-to-object-storage-hive.mdx

```diff
@@ -16,13 +16,12 @@ dates:
 
 ## Overview
 
-Since the 27th of January 2022 the [Object Storage](/storage/object/quickstart/) platform has been migrated to our internally developed backend: [Hive](https://papers.s3.fr-par.scw.cloud/hive.pdf).
-We use the robust Erasure Code (6+3) to protect the data stored on Scaleway Object Storage. This mechanism now moves from one datacenter to three Availability Zones (AZ), which means that the 6+3 fragments will be spread over three independent Availability Zones in one region.
+Scaleway Object Storage uses a robust Erasure Code (6+3) to protect data stored on our infrastructure. This mechanism splits your data over Availability Zones (AZ), which means that the 6+3 fragments will be spread over three independent Availability Zones in one region.
 
-In summary, the main advantage of Multi-AZ is that it ensures the resilience of your data storage, even in the event of a total loss of an AZ.
-Scaleway Object Storage is built entirely on Scaleway's technical stack, which guarantees true sovereignty at all levels, from infrastructure to software. Thanks to our new backend, you can store millions of objects in a single bucket and scale your infrastructure, without friction and without limit. 
+Multi-AZ ensures the resilience of your data storage, even in the event of a total loss of an AZ.
+Scaleway Object Storage is built entirely on Scaleway's technical stack, which guarantees true sovereignty at all levels, from infrastructure to software. You can store millions of objects in a single bucket and scale your infrastructure, without friction and without limit.
```

Comment:
```suggestion
Scaleway Object Storage is built entirely on Scaleway's technical stack, which guarantees true sovereignty at all levels, from infrastructure to software. You can store millions of objects in a single bucket and scale your infrastructure without friction or limit.
```

Reply:
This file will be deleted soon so I did not apply the changes